LayoutsCodelab/app/src/main/java/com/codelab/layouts/ConstraintLayout.kt

```diff
@@ -54,7 +54,7 @@ fun ConstraintLayoutContent() {
 @Composable
 fun DecoupledConstraintLayout() {
     WithConstraints {
-        val constraints = if (minWidth < 600.dp) {
+        val constraints = if (maxWidth < maxHeight) {
```

Comment:
Just double checking, do this work with the codelab text? (or are you editing that?)

Reply:
I updated the codelab code snippet but the text doesn't mention it.
I had to change it as minWidth is always returning 0, on portrait or landscape, which made this behaviour not working at all